src/main/java/com/example/demo3/model/User.java

```diff
@@ -17,7 +16,7 @@ public class User
     private String phone;
     private String email;
     private String city;
-    private Integer is_auth;
+    private boolean isAuth;
```

Comment:
Keep naming variables according javaguidelines. "_" can use in constant e.g. MY_LOGGER_MESSAGE